### PR TITLE
PFRS Step 6 and Step 8

### PR DESCRIPTION
**Pull request by dhaselhan**

### CODE REVIEW

File: portal-frontend/src/app/features/alcs-edit-submission/alcs-edit-submission.component.ts [hunk 1]
```diff
@@ -189,6 +191,9 @@ export class AlcsEditSubmissionComponent implements OnInit, OnDestroy, AfterView
         if (this.profoProposalComponent) {
```
Comment (mhuseinov) [inline portal-frontend/src/app/features/alcs-edit-submission/alcs-edit-submission.component.ts:191]: looks like a good time to move EditApplicationSteps.Proposal into separate function to make it easier to read the code

File: portal-frontend/src/app/features/edit-submission/edit-submission.component.ts [hunk 2]
```diff
@@ -197,6 +199,9 @@ export class EditSubmissionComponent implements OnInit, OnDestroy, AfterViewInit
         if (this.pofoProposalComponent) {
```
Comment (mhuseinov) [inline portal-frontend/src/app/features/edit-submission/edit-submission.component.ts:199]: same as previous, just extracting into function will be good

File: portal-frontend/src/app/features/edit-submission/proposal/pfrs-proposal/pfrs-proposal.component.html [hunk 3]
```diff
@@ -0,0 +1,471 @@
...
+            >No
+          </mat-button-toggle>
+        </mat-button-toggle-group>
+      </div>
+    </div>
+
+    <div class="full-row">
+      <label class="subheading2" for="noticeOfWork">Notice of Work</label>
+      <app-info-banner>
+        Please refer to <a>TODO: Removal of Soil</a> on the ALC website for more information
```
Comment (mhuseinov) [inline portal-frontend/src/app/features/edit-submission/proposal/pfrs-proposal/pfrs-proposal.component.html:444]: Do we have a ticket to fill up this links?

**Review by mhuseinov — APPROVED** [review]
🚀

File: portal-frontend/src/app/features/edit-submission/proposal/pfrs-proposal/pfrs-proposal.component.html (hunk 3 above)
Comment (dhaselhan) [reply portal-frontend/src/app/features/edit-submission/proposal/pfrs-proposal/pfrs-proposal.component.html:444]: I believe so, this is pending on the ALC launching their new website though which is TBD

File: portal-frontend/src/app/features/alcs-edit-submission/alcs-edit-submission.component.ts (hunk 1 above)
Comment (dhaselhan) [reply portal-frontend/src/app/features/alcs-edit-submission/alcs-edit-submission.component.ts]: Good call, done!

File: portal-frontend/src/app/features/edit-submission/edit-submission.component.ts (hunk 2 above)
Comment (dhaselhan) [reply portal-frontend/src/app/features/edit-submission/edit-submission.component.ts]: Good call, done!

**Pull request merged**